src/harness/fakesHosts.ts

```diff
@@ -32,6 +32,11 @@ namespace fakes {
             this._env = env;
         }
 
+        getWidthOfTerminal(): number {

```

Comment:
Make this private field instead that is either undefined or returns  testTerminalWidth 
```ts
const testTerminalWidth = Number.parseInt(this.getEnvironmentVariable("TS_TEST_TERMINAL_WIDTH"));
this.getWidthOfTerminal = Number.isNaN(testTerminalWidth) ? undefined : () => testTerminalWidth;
```